0000-template.md

```diff
@@ -22,9 +22,6 @@ familiar with the project to understand, and for somebody familiar with the
 implementation to implement. This should get into specifics and corner-cases,
 and include examples of how the feature is used. Any new terminology should be
 defined here.
-Use definitive language, rather than proposing ideas here. The content of this
-RFC can be discussed and refined on GitHub. Any unanswered questions can also
-be written in the final 'Unresolved questions' section.
```

Comment:
Have these lines intentionally been removed? If so - what is the thinking?

Reply:
It was unintentional.  I moved the default file when I should have copied it.